### DISCUSSION TITLE
Add suffix options for generated classes

### DISCUSSION THREAD

**Discussion by fangliml** [square/wire#2778] [Ideas]
I'm integrating Wire in a very large and old codebase wich already uses protoc. It would be awesome to have the option of adding a suffix to the generated classes (not only the services), to avoid conflicts with old classes.

**Comment by oldergod** [comment]
Currently, the only thing doable to differentiate them would be to use the `wire_package` file option which will tell Wire to override the java_package or proto package when generating files.
